python/ray/serve/tests/test_controller.py

```diff
@@ -148,6 +148,7 @@ def autoscaling_app():
                     "message": "",
                     "last_deployed_time_s": deployment_timestamp,
                     "deployed_app_config": None,
+                    "source": 1,
```

Comment:
This feels awkward to me - can the enum be a https://docs.python.org/3/library/enum.html#enum.StrEnum instead? Or perhaps if they need to be `int`s we'd use the existing enum field names to convert at the last moment, just for the API (not internally)?

Reply:
Yes, I think it would be better to change this to a string enum, now that it's exposed through the rest api.